rust/rope/src/engine.rs

```diff
@@ -85,28 +85,22 @@ impl Engine {
         None
     }
 
+    /// Get the contents of the document at a given revision number
     fn get_rev_from_index(&self, rev_index: usize) -> Rope {
```

Comment:
I think this function might be another candidate for a renaming. Maybe something like `get_revision`? or `document_at_revision`? I think the word `from` is confusing, given the way it is used in Rust to indicate conversion constructors. Maybe `for` would be more communicative? I also don't love `get` for something that isn't a getter. 😕